src/components/ha-assist-chat.ts

```diff
@@ -498,7 +496,7 @@ export class HaAssistChat extends LitElement {
         bottom: 0px;
         right: 0px;
         left: 0px;
-        padding: 24px;
+        padding-bottom: 16px;
```

Comment:
The x padding added a space betweren the chat bubbles and the scrollbar. Now it sticks to the scrollbar

Reply:
Ugh good point.

Reply:
Fixed.